### PR TITLE
Update wasm.mdx

### PR DESCRIPTION
**Pull request by Thoralf-M**
# Description of change

Link to wasm readme because the nodejs examples are wrong

## Type of change
- Documentation Fix

### CODE REVIEW

**Review by Thoralf-M — COMMENTED** [review]
Can't approve since it's my own PR, but looks good

**Comment by lucas-tortora** [comment]
https://iota-wiki-b12pauybs-iota1.vercel.app/iota-sdk/getting-started/wasm/

**Comment by lucas-tortora** [comment]
@Thoralf-M , whenever you have a minute, please review the changes I added. Thanks!

**Review by Dr-Electron — APPROVED** [review]
LGTM

File: docs/build/iota-sdk/1.0/docs/getting-started/wasm.mdx [hunk 1]
```diff
@@ -147,51 +156,69 @@ A bundler such as [webpack](https://webpack.js.org/) or [rollup](https://rollupj
...
+that connects to the [Shimmer Testnet](https://api.testnet.shimmer.network) using the
+[`MnemonicSecretManager`](https://wiki.iota.org/iota-sdk/references/nodejs/interfaces/MnemonicSecretManager/)
+by calling
+the [`Wallet.createAccount(data)`](https://wiki.iota.org/iota-sdk/references/nodejs/classes/Wallet/#createaccount)
+function.
+
+
+<Tabs groupId="usage" queryString>
+<TabItem value="Node.js" label="Node.js">
+<div className={'hide-code-block-extras'}>
```
Comment (Dr-Electron) [inline docs/build/iota-sdk/1.0/docs/getting-started/wasm.mdx:201]: Do we want to compare with node.js code here? IF so, was/web should be first and default tho @lucas-tortora

**Review by Dr-Electron — CHANGES_REQUESTED** [review]
Nevermind. Changed my mind :trollface:

File: docs/build/iota-sdk/1.0/docs/getting-started/wasm.mdx (hunk 1 above)
Comment (lucas-tortora) [reply docs/build/iota-sdk/1.0/docs/getting-started/wasm.mdx:201]: I don't think I understood what you mean. Why should Web be first?

**Pull request merged**